packages/datagateway-download/src/downloadApi.ts

```diff
@@ -361,10 +361,11 @@ export const getDatafileCount: (
       .get<number>(`${settings.apiUrl}/datafiles/count`, {
         params: {
           where: {
-            DATASET_ID: {
+            'dataset.id': {
               eq: entityId,
             },
           },
+          include: 'dataset',
```

Comment:
Adding this `include: dataset` causing this function to fail on Diamond prod - it's giving an error to do with ICAT not supporting multiple attributes? Regardless, the `where` on it's own works fine for the `python-icat` backend (although I presume not for the database backend)

Can we investigate this using DLS preprod to double check?

Reply:
Does Diamond prod have ICAT Server 4.11.1? That might be why there's an issue

Reply:
It doesn't no - in fact I forgot that currently lots of requests currently fail due to this! So it's not a DataGateway issue then - perhaps we might need to use an older version of datagateway-api for DLS if the ICAT 4.11.1 release is delayed

Reply:
I could certainly find the most recent API commit not to require 4.11.1 if needed, but that'll mean the distinct filter will be broken on that version of the API, something which Diamond need I think?

Reply:
This is more thinking of a worst case scenario in which we develop a work around in DataGateway to the issue (similar to what TopCAT does) then in that case we'd also need to drop the datagateway-api to a previous commit yeah. Nothing to worry about now though hopefully!

Reply:
Yeah if we need to do something like that, just let me know and I'm sure we can get it sorted :)